AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -31,6 +39,7 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
         self.dismissViewControllerAnimated(true, completion: nil)
         self.image = info[UIImagePickerControllerOriginalImage] as! UIImage
         imageView.image = self.image
+        self.data = self.compressFile() as? NSData
```

Comment:
Why are we compressing this? We probably want the image uploaded to be the same as the image the user sees on the screen... and, it doesn't look this compresses (or operates on a file), it actually just reduces image quality, if we need it could rename to "scaleDownImage".